rakefile

```diff
@@ -146,6 +218,13 @@ rototiller_task :acceptance do |t|
   Rake::Task["performance"].execute
 end
 
+desc 'Run Performance setup and a very short iteration of gatling tests locally'
```

Comment:
this looks like another opportunity to split out via separate commit

Reply:
Hmm, looks like I copied that task and was going to modify it, but ended up making things work a different way, I'm removing this.